sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -24,7 +24,8 @@
 
 
 class DefaultAzureCredential(ChainedTokenCredential):
-    """A default credential capable of handling most Azure SDK authentication scenarios.
+    """A credential capable of handling most Azure SDK authentication scenarios. See
+    https://aka.ms/azsdk/python/identity/credential-chains#usage-guidance-for-defaultazurecredential.
```

Comment:
Can you also mirror this in the [async counterpart ](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/azure/identity/aio/_credentials/default.py), as well?